attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/AttributeServiceImpl.java

```diff
@@ -190,40 +191,50 @@ public void delete(
       return;
     }
 
-    Iterator<Document> documents =
-        collection.search(getQueryForFilter(tenantId.get(), modifiedRequest));
-    boolean status =
-        StreamSupport.stream(Spliterators.spliteratorUnknownSize(documents, 0), false)
-            .map(Document::toJson)
-            .map(
-                attrMetadataDoc -> {
-                  try {
-                    AttributeMetadata metadata =
-                        AttributeMetadataModel.fromJson(attrMetadataDoc).toDTO();
-                    boolean response =
-                        collection.delete(AttributeMetadataDocKey.from(tenantId.get(), metadata));
-                    if (!response) {
+    try {
+      Iterator<Document> documents =
+          collection.search(getQueryForFilter(tenantId.get(), modifiedRequest));
+      boolean status =
+          StreamSupport.stream(Spliterators.spliteratorUnknownSize(documents, 0), false)
```

Comment:
nit -` Streams.stream(documents)` (since we already are using guava)

Edit: realized this just got wrapped and was pre-existing, don't need to fix